server/android_notification_server.go

```diff
@@ -62,28 +62,30 @@ func (me *AndroidNotificationServer) SendNotification(msg *PushNotification) Pus
 		}
 	}
 
-	regIDs := []string{msg.DeviceId}
-	gcmMsg := gcm.NewMessage(data, regIDs...)
-
-	sender := &gcm.Sender{
-		ApiKey: me.AndroidPushSettings.AndroidApiKey,
-		Http:   httpClient,
```

Comment:
Do we need to pass the http client to the new library somehow to use the timeouts and such we have specified [here](https://github.com/mattermost/mattermost-push-proxy/blob/4fcc302a9dd0d1fc5449c77550c5d4cb79ceef68/server/push_notification.go#L30)?

Reply:
Not really it uses it internally, I think we can can remove it, I’ll check if its needed for ios

Reply:
@hmhealey removed

Reply:
Does the built-in one have the correct timeouts and such though? I assume there was a reason we specific our own client as opposed to just using the default one from `net/http`

Reply:
you can take a look at the library.. the previous one required an httpClient, this one does not

Reply:
Taken from the library

```go
const (
	// DefaultEndpoint contains endpoint URL of FCM service.
	DefaultEndpoint = "https://fcm.googleapis.com/fcm/send"

	// DefaultTimeout duration in second
	DefaultTimeout time.Duration = 30 * time.Second
)
```

Reply:
Ah okay. Makes sense then